packages/uswds-core/src/styles/tokens/color/_black-transparent.scss

```diff
@@ -1,14 +1,30 @@
+/**
+ * Do not edit directly
+ * Generated on Tue, 22 Oct 2024 19:52:21 GMT
+ */
```

Comment:
**note:** We can omit these file headers by updating our config:

```diff
        format: "scss/map-deep",
        options: {
          themeable: false,
+         showFileHeader: false
        },
```

Reply:
Custom file headers resolved in [7673fa5](https://github.com/uswds/uswds/pull/5898/commits/7673fa574f4c65c862e3facdfa2043cfdc984eea)